pilot/pkg/model/context.go

```diff
@@ -561,7 +561,7 @@ const (
 	NodeMetadataRouterMode = "ROUTER_MODE"
 
 	// NodeMetadataInstanceIPs is the set of IPs attached to this proxy
-	NodeMetadataInstanceIPs = "INSTANCE_IPS"
+	NodeMetadataInstanceIPs = "ISTIO_META_INSTANCE_IPS"
```

Comment:
Do we do the stripping in bootstrap, which is where we add this to the metadata with the prefix, so this one never gets stripped?

If so, maybe a better fix is to change the bootstrap to not have the prefix, like @hzxuzhonghu said

afaik in the final config nothing should have ISTIO_META